.github/workflows/pre-submit.cli.yml

```diff
@@ -36,4 +36,5 @@ jobs:
       - name: Run tests for verifier
         run: |
           set -euo pipefail
+          make unit-test
```

Comment:
Do we need this change? Unit tests are run in another workflow, no?

Reply:
I'm not 100% but it seems like the regression-tests command does cover the unit tests, as well.

* [go test -mod=vendor -tags=regression ...](https://github.com/ramonpetgrave64/slsa-verifier/blob/edb1decd90fa511993ead9c3f7917e5abe371c10/Makefile#L47)
* [go:build](https://github.com/ramonpetgrave64/slsa-verifier/blob/edb1decd90fa511993ead9c3f7917e5abe371c10/cli/slsa-verifier/main_regression_test.go#L1)